bin/install-ci-dependencies.py

```diff
@@ -173,15 +173,15 @@ def unpack(self):
 
 def get_android_url():
   if os.getenv('TRAVIS'):
-    if os.getenv('TOOLCHAIN') == 'android-ndk-r10e-api-19-armeabi-v7a-neon':
```

Comment:
Both NDK versions can be supported simply by not removing old code, just keep checking toolchain name